src/main/java/emt/gthandler/common/loader/EMT_GT_Loader.java

```diff
@@ -40,6 +43,8 @@ public void run() {
 
         LEG = new EMT_Large_Essentia_Gen(aIDoffset + TIERS + 1, "Large Essentia Generator", "Large Essentia Generator").getStackForm(1L);
 
+        //Dragonblood recipe for magics haters
```

Comment:
Same here. Also use `FluidRegistry#getFluid` instead of `#getFluidID` for consistency

Reply:
oh it's actually typing fail lol